src/__tests__/Geolocation.test.js

```diff
@@ -0,0 +1,5 @@
+//TODO - add testing for geolocation finding/tracking... some how!
```

Comment:
Can we please delete this file since it is empty? We can add tests in another PR :)